src/attributes.jl

```diff
@@ -356,6 +356,7 @@ struct VariableName <: AbstractVariableAttribute end
 An initial assignment of the variables that the solver may use to warm-start the solve.
 """
 struct VariablePrimalStart <: AbstractVariableAttribute end
+mustcopy(::VariablePrimalStart) = false
```

Comment:
Now this is tricky. For nonconvex problems, I consider the starting point an essential part of the problem definition, since the combination of the starting point and algorithm define the solution. Let's not set `mustcopy` to `false` here until it becomes an issue. We may eventually need to define two different attributes to capture the two different meanings of starting points.

Reply:
Done. Does is also old for ConstraintPrimalStart and ConstraintDualStart ?